Microsoft Graph Mail Enhancement (#23041)
demisto/content: Packs/MicrosoftGraphListener/ReleaseNotes/1_3_0.md

#### Integrations
##### Microsoft Graph Mail Single User
- Updated the Docker image to: *demisto/crypto:1.0.0.40696*.

> - Added a "Display full email body" integration parameter for getting the full email body instead of just a preview in fetch-incidents.

Reviewer: ```suggestion
- Added a "Display full email body" integration parameter to fetch incidents with the full email body and not just a preview.
```
